CHANGES.rst

```diff
@@ -2076,6 +2076,9 @@ assign_wcs
 
 - fix input units to meters when filter=OPAQUE [#2134]
 
+- A ``ValueError`` is now raised if input data is missing ``xref_sci`` or ``yref_sci`` keywords. [#4561]
```

Comment:
It looks as if you added these notes near the end of CHANGES.rst.  They should go near the beginning of the file, under the heading "0.14.3 (Unreleased)".

Reply:
Whoops! Moved the notes